### ISSUE TITLE
Try Linzen task with binary cross entropy.

### ISSUE THREAD

**Issue by viking-sudo-rm** [viking-sudo-rm/StackNN#19]
The original Linzen paper used binary cross entropy instead of softmax cross entropy.

https://github.com/TalLinzen/rnn_agreement/blob/master/rnnagr/agreement_acceptor.py

Results are not replicating with softmax cross entropy. We can try switching to binary cross entropy.
